open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/TermFVT.java

```diff
@@ -135,7 +135,7 @@ public void run() throws SubjectAreaFVTCheckedException, InvalidParameterExcepti
             throw new SubjectAreaFVTCheckedException("ERROR: Governance actions retention not returned  as expected");
         }
         if (!governanceActions.getCriticality().getLevel().equals(term3.getGovernanceActions().getCriticality().getLevel())) {
-            throw new SubjectAreaFVTCheckedException("ERROR: Governance actions criticality not returned  as expected");
+            throw new SubjectAreaFVTCheckedException("ERROR: Governance actions criticality not returned  as expected. ");
```

Comment:
Minor: string still contains a superfluous space.

Reply:
> Minor: string still contains a superfluous space.

I suggest that I fix if I need to make another code change on this pr.